src/components/CopyToClipboard/copyText.ts

```diff
@@ -1,11 +1,43 @@
 export function copyText(text: string) {
     if (navigator?.clipboard?.writeText) {
-        try {
```

Comment:
This try..catch was added in case of NotAllowed exception, can we return try catch?